src/libertem/web/base.py

```diff
@@ -104,3 +106,21 @@ async def send_existing_job_results(self):
             await self.send_results(
                 result_set, job_id, analysis_id, details, finished=True
             )
+
+def handle_session(reqHandler, state, event_registry):
```

Comment:
This is a function I added to handle both the websocket and normal middle-ware. It checks if there is a SESSID cookie and sets up a random id if its not found.